packages/data-stores/src/onboard/actions.ts

```diff
@@ -455,6 +455,11 @@ export const setHideFreePlan = ( hideFreePlan: boolean ) => ( {
 	hideFreePlan,
 } );
 
+export const setHidePlansFeatureComparison = ( hidePlansFeatureComparison: boolean ) => ( {
+	type: 'SET_HIDE_PLANS_FEATURE_COMPARISON' as const,
+	hidePlansFeatureComparison,
+} );
```

Comment:
Yes, we created it to manage onboarding in the new flows. Added [this issue](https://github.com/Automattic/wp-calypso/issues/75954) to add a README.

I feel like onboard is a better candidate to this, as it manages all the things related to the experience of going through the flow!

Reply:
Cool. No objection.

I think it might be a little weird if `PlanFeaturesMain` ends up querying the onboard store to figure out whether to render a comparison grid or not - if, say, `hidePlansFeatureComparison` is no longer a prop but a hook (something like `useGridVisibilities()`) that queries a store to determine which grids to show.